### DISCUSSION TITLE
System requirements Vorta for Mac ??

### DISCUSSION THREAD

**Discussion by MacH59-cos** [borgbase/vorta#1247] [General]
Looking for System requirements Vorta for Mac. Can not find anything in that regard. Tried to install Vorta 0.82 on a Mac with 10.10.5 but it failed. Started to look for information which version I would need to use on 10.10.5 but as I mentioned before I could not find anything.

Is it possible to use Vorta with such an old version or maybe even older versions of macOS??

Any information is highly appreciated, but a nice list would be great.

**Comment by m3nu** [comment]
The packaged macOS version will only work on the current macOS version and maybe the one before. Depends on the version Github Actions uses.

If it doesn't work, you can still install via PyPi, which should work on most versions.
